src/extension/abstract.js

```diff
@@ -21,16 +25,20 @@ export class AbstractExtension {
     this._type = value
   }
 
+  // Content.
+
   /**
-   * The content of this extension.
+   * The Custom XML object.
+   *
+   * Bounded: 1
    *
-   * @type {Element}
+   * @type {String}
```

Comment:
This is still an `Element`, right? Since extensions are freeform, we don't want to convert them to our own model, but since we've generally already parsed them to XML, we're exposing them as such.

I don't mind renaming `.content` to `.xml` if you think it's clearer, although then, I'd maybe go with `.element` or `.xmlElement`.

One thing I don't really like in the current version is that `.content` actually gives you the `<Extension>` node. The reasoning is that an `<Extension>` can have multiple XML children, so to support that, an instance of `Extension` would have to have `.content` of the type `Node[]`, effectively identical to `originalElement.children`. But that seemed even more confusing. Perhaps we should just mimic the XML API and have a `.children` property on `Extension`?

Reply:
I have not updated this. Same behavior as before.

Reply:
You mean you haven't changed it in the parser, right? In that case,

1. The type on line 27 would need to change back to `Element`.

2. But I'd actually change it from `content: Element` to `childNodes: Element[]` as described above. (Technically `NodeList` but an array is going to be easier to work with.)

Reply:
Yes, parser has not change. So I will update the comment and the parser. Thanks.